CHANGELOG.md

```diff
@@ -13,9 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
   - Update Gradle to `7.1.1` version
   - Update Shadow plugin to `7.0.0` version
   - Update Kotlin to `1.5.21` version
+  - Rethink format task approach ([issue: $306](https://github.com/JLLeitschuh/ktlint-gradle/issues/306))
 
 ### Fixed
   - Pre-commit hook causing conflicts ([issue: #443](https://github.com/JLLeitschuh/ktlint-gradle/issues/443)) ([#502](https://github.com/JLLeitschuh/ktlint-gradle/pull/502))
+  - `ktlintFormat` create empty directories in `src/` dir ([issue: $423](https://github.com/JLLeitschuh/ktlint-gradle/issues/423))
```

Comment:
Is this syntax correct? Shouldn't it be this?
```suggestion
  - `ktlintFormat` create empty directories in `src/` dir ([issue: #423](https://github.com/JLLeitschuh/ktlint-gradle/issues/423))
```
Same question about the line above

Reply:
Fixed the typo!